server/model/splittable/split_table_recognizer.py

```diff
@@ -0,0 +1,132 @@
+import pickle
+
+from server.service.data.data import *
+
+
+class SplitTableRecognizer:
+    def process(self, document: Document) -> Document:
+        pass
```

Comment:
Jeśli to jest abstrakcyjna metoda, to najlepiej tu dać `raise NotImplementedError()`, żeby wymusić implementację tego w klasach dziedziczących